src/en/screen-reader.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Screen Reader
+---
+A screen reader is an assistive technology that translates visual content into audio content or touch content for people with difficulty seeing. 
+
+Assistive technology supports the lives of people with disabilities by supporting their functional capability within the spheres of work, learning, and recreation. 
```

Comment:
This sentence is great, but I think it might be more appropriate in a definition for "assistive technology." I think in this space here, I'd expect to see some further explanation of the functionality of screen readers in particular.